src/engraving/style/styledef.cpp

```diff
@@ -224,7 +224,7 @@ const std::array<StyleDef::StyleValue, size_t(Sid::STYLES)> StyleDef::styleValue
     { Sid::articulationAnchorLuteFingering, "articulationAnchorLuteFingering", int(ArticulationAnchor::BOTTOM_CHORD) },
     { Sid::articulationAnchorOther, "articulationAnchorOther", int(ArticulationAnchor::TOP_STAFF) },
     { Sid::articulationStemHAlign,  "articulationStemHAlign",  int(ArticulationStemSideAlign::AVERAGE) },
-    { Sid::articulationKeepTogether, "articulationKeepTogether", true },
+    { Sid::articulationKeepTogether, "articulationKeepTogether", 1 },
```

Comment:
Why are you making it an `int` instead of just `bool`? If the only reason is to get the EditStyle UI working, there must be a better solution for that. For example, adding handling for combo boxes for boolean properties in `EditStyle::setValues()` and `EditStyle::getValue()`.